homestar-runtime/src/settings.rs

```diff
@@ -76,12 +76,16 @@ pub struct Network {
     /// [Swarm]: libp2p::swarm::Swarm
     #[serde(with = "http_serde::uri")]
     pub(crate) listen_address: Uri,
-    /// Mdns IPv6 enable flag.
+    /// Enable Rendezvous protocol.
+    pub(crate) enable_rendezvous: bool,
+    /// Enable mDNS.
+    pub(crate) enable_mdns: bool,
+    /// MDNS IPv6 enable flag
     pub(crate) mdns_enable_ipv6: bool,
-    /// Mdns query interval.
+    /// MDNS query interval.
     #[serde_as(as = "DurationSeconds<u64>")]
     pub(crate) mdns_query_interval: Duration,
-    /// Mdns TTL.
+    /// MDNS TTL.
```

Comment:
not a big deal, but should these be `mDNS` instead?

Reply:
Yeah, good question! Was a bit conflicted because we capitalize the first letter of the comments for the other settings, but it doesn't make as much sense for this one.

Reply:
yeah, probably not a big deal either way haha. It looks like all the comments in this file capitalize the first letter, but some comments in other files on the PR don't 🤷🏼 I'd vote for `mDNS`, but not married to either approach!

Reply:
@bgins for a proper noun, ok for it to be `mDNS`.